src/snowflake/snowpark/modin/plugin/compiler/snowflake_query_compiler.py

```diff
@@ -13096,7 +13101,7 @@ def output_col(
 
             if np.isnan(n):
                 # Follow pandas behavior
-                return pandas_lit(np.NaN)
+                return pandas_lit(np.NaN)  # type: ignore
```

Comment:
I don't think you need this anymore since the numpy fix is merged - rebase and see if that is the case.